go/core/action.go

```diff
@@ -64,7 +64,18 @@ type Action[In, Out, Stream any] struct {
 	Metadata    map[string]any
 }
 
-// See js/common/src/types.ts
+// See js/core/src/action.ts
+
+// DefineAction creates a new Action and registers it.
+func DefineAction[In, Out any](name string, atype ActionType, metadata map[string]any, fn func(context.Context, In) (Out, error)) *Action[In, Out, struct{}] {
```

Comment:
Similarly `DefineStreamingAction` as well for https://github.com/firebase/genkit/blob/main/go/ai/generator.go#L69-70
?

Reply:
Will do.